tests/test-pg-error-view.json

```diff
@@ -0,0 +1,61 @@
+{
+    "webserver": {
+        "views/test-pg-error": {
+            "view": "fostgres.control.pg-error",
+            "configuration": {
+                "execute": {
+                    "view": "fostgres.sql",
+                    "configuration": {
+                        "dbname": [
```

Comment:
These are no longer needed as what you have I think is the default anyway.

Reply:
Removed